stylesheets/stylesheet.css

```diff
@@ -477,6 +477,33 @@ Custom Styles
   margin-bottom: 0;
 }
 
+.projects .label .count {
+  background: none white;
+  border: 1px solid white;
+  border-radius: 8px;
+  color: #0090ff;
+  display: inline-block;
+  height: 16px;
+  line-height: 16px;
+  margin: -2px -4px 0 4px;
```

Comment:
Not sure what the `-2px` here is for, but this looks weird in Chrome

![screen shot 2015-02-09 at 8 42 32 pm](https://cloud.githubusercontent.com/assets/359239/6103840/48c84774-b09c-11e4-94ae-29dceb0e7cde.png)

Putting this back to `0px` feels better:

![screen shot 2015-02-09 at 8 42 06 pm](https://cloud.githubusercontent.com/assets/359239/6103850/5783e426-b09c-11e4-9ef9-5253da6673f0.png)

The text also seems to be slightly above the vertical center, but I Am Not A CSS Guru...